calour/util.py

```diff
@@ -43,6 +43,44 @@
 logger = getLogger(__name__)
 
 
+def _iter_fasta(file_name):
```

Comment:
not sure this should be private or public. but it definitely should be in io.py - could u move it? 

and could you rename `file_name` to `fp` to be consistent with other functions.

Reply:
file_name -> fp : fixed
agree more logical in io.py, but then we get cyclic import problem (io.py imports amplicon_experiment.py (for read() ), and amplicon_experiment.py needs _iter_fasta() for the filter_fasta() command). can think about moving filter_fasta() from amplicon_experiment.py, but it is specific for AmpliconExperiment (not a generic calour.Experiment since assumes features are sequences), so problem moving it to filtering.py.... Any idea how to solve?

Reply:
let me check the cyclic import issue.